### PR TITLE
feat(route): add Lever HRIS

### PR DESCRIPTION
**Pull request by tsarlewey**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/lever/lever
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — CHANGES_REQUESTED** [review]
Now route should follow [v2 standard](https://docs.rsshub.app/en/joinus/script-standard.html)

**Comment by TonyRL** [comment]
Please also provide routes with actual parameter(s) for testing.

File: lib/v2/lever/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        // the source description
+        description: `Auto Generated RSS Spec for ${domain}`,
+        // iterate through all leaf objects
+        item: data.map((item) => ({
+            // the article title
+            title: item.text,
+            // the article content
+            description: item.descriptionPlain,
+            // the article publish time
+            pubDate: parseDate(new Date(item.createdAt)),
```
Comment (TonyRL) [inline lib/v2/lever/index.js:27]:
```suggestion
            pubDate: parseDate(item.createdAt),
```

File: lib/v2/lever/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/lever/:domain': ['tsarlewey'],
```
Comment (TonyRL) [inline lib/v2/lever/maintainer.js:2]:
```suggestion
    '/:domain': ['tsarlewey'],
```

File: lib/v2/lever/router.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/lever/:domain', require('./index.js'));
```
Comment (TonyRL) [inline lib/v2/lever/router.js:2]:
```suggestion
    router.get('/:domain', require('./index.js'));
```

**Review by TonyRL — COMMENTED** [review]
It seems lib/v2/lever/templates/author.art and lib/v2/lever/templates/description.art aren't actually used in your code.

File: lib/v2/lever/router.js (hunk 3 above)
Comment (tsarlewey) [reply lib/v2/lever/router.js:2]: Thank you @TonyRL for the suggested changes - and I apologize for alerting you, I thought I was committing to a branch. I also deeply appreciate your suggestions and have tested them and found them to have fixed my route(s). 

One question I have is your comment about:

"
Please also provide routes with actual parameter(s) for testing.
"

Do you mean in other.md provide example routes that people could see an example feed? E.g. /lever/lever (as it is now)

Comment (TonyRL) [reply lib/v2/lever/router.js:2]: Put `/lever/lever` into the \`\`\`routes\`\`\` region in your PR description for automated testing

File: docs/en/other.md [hunk 4]
```diff
@@ -248,3 +248,7 @@ Type
 ### User
 
 <RouteEn author="SettingDust Halcao" example="/uraaka-joshi/_rrwq" path="/uraaka-joshi/:id" :paramsDesc="['User ID']" radar="1" rssbud="1"/>
+
+### Lever
```
Comment (TonyRL) [inline docs/en/other.md:252]:
```suggestion
## Lever

### Recruitment
```
You also need a H3 heading

Comment (tsarlewey) [reply docs/en/other.md:252]: So I do...

**Comment by tsarlewey** [comment]
@TonyRL Is it a requirement to have the /templates folder but not a requirement to use it? Per the documentation: 
https://docs.rsshub.app/en/joinus/script-standard.html

image

**Comment by TonyRL** [comment]
> @TonyRL Is it a requirement to have the /templates folder but not a requirement to use it? Per the documentation: https://docs.rsshub.app/en/joinus/script-standard.html
> 
> image

You only need templates if you want to render custom HTML other than what is provided from API/HTML

File: lib/v2/lever/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    const response = await got({
+        method: 'get',
+        url: `https://api.lever.co/v0/postings/${domain}?mode=json`,
+    });
+    const data = response.data;
+    ctx.state.data = {
+        // the source title
+        title: `${domain}'s Job feed in RSS`,
+        // the source link
+        link: `https://api.lever.co/v0/postings/${domain}?mode=json`,
```
[the remark this answers is not in the document]
Comment (tsarlewey) [reply lib/v2/lever/index.js]: In that case, it would be the HTML readable site: https://api.lever.co/v0/postings/${domain}

**Pull request merged**